browsers.js

```diff
@@ -90,6 +90,7 @@ export async function createBrowserWorker( url, browser, options, restarts = 0 )
 		const driver = await createDriver( {
 			browserName: browser.browser,
 			headless,
+			safariTp: options.safariTp,
```

Comment:
I think we could destructure `safariTp` above and pass the same as `headless`, just for consistency..